src/docker/docker-compose.yml

```diff
@@ -2,21 +2,23 @@ version: '3'
 
 services:
   gitlab:
-    image: 'gitlab/gitlab-ce:14.7.0-ce.0'
+    image: 'gitlab/gitlab-ce:latest'
     restart: "no"
     hostname: 'localhost'
     ports:
-      - '10080:80'
-      - '10022:22'
+      - '55580:80'
+      - '55522:22'
+      - '55443:443'
     volumes:
-      - '/srv/docker/gitlab/config:/etc/gitlab'
-      - '/srv/docker/gitlab/logs:/var/log/gitlab'
-      - '/srv/docker/gitlab/data:/var/opt/gitlab'
+      -  '/Users/harshpratapsingh/srv/docker/gitlab/config:/etc/gitlab'
```

Comment:
I would suggest not to change the paths in this file, because these are also shared with users of other operating systems such as Linux.

Reply:
```suggestion
      -  '/srv/docker/gitlab/config:/etc/gitlab'
```